### DISCUSSION TITLE
Be able to see more than the path's filename in the preview title

### DISCUSSION THREAD

**Discussion by mangelozzi** [ibhagwan/fzf-lua#1330] [Ideas]
Sometimes when I use the fuzzy line search, which is awesome, due to the vertical sliced architecture of django app design, where you have the same name files inside each app, e.g.:
```
+ books (app)
|      +---views.py
|      +---models.py
|
+ posts (app)
|      +---views.py
|      +---models.py
|
+ polls (app)
      +---views.py
      +---models.py
```
Which means one might have a lot of matches in files all names `views.py`, which does not mean too much. To be able to see `polls/views.py` means a lot more than just `views.py`. Does anyone else encounter that issue?

Here you can see I tried the horizontal split to hopefully see more of the file path in the picker, but it still shows (understandabley so) the filename (`views.py): 
![image](https://github.com/user-attachments/assets/2686143d-b29c-4608-b440-092fe94d455e)

Under preview I could not see any options for how paths are shown in their title:
```
      title              = true,               -- preview border title (file/buf)?
      title_pos      = "center",        -- left|center|right, title alignment     
```

Not a big thing, just wondering if anyone else has encountered this or maybe it's just me.

**Comment by mangelozzi** [comment]
And just to confirm, while looking into this I see the `lines` picker is matching the path, in addition to lines. Here you can see it matched `serializer` from the filename, but the line in the buffer does not contain the patter `serializer`:

![image](https://github.com/user-attachments/assets/80b52dab-e587-4bd7-ac25-733d87a70020)

**Reply by ibhagwan** [reply]
I personally also like to see the full path in the preview title, use `previewers.builtin.title_fnamemodify` to control the behavior of the title, my configuration:
https://github.com/ibhagwan/nvim-lua/blob/64812566ae22fac5eca742a46ad92a813b746bb9/lua/plugins/fzf-lua/setup.lua#L138-L141:
```lua
  previewers = {
    bat = { theme = "Coldark-Dark", args = "--color=always --style=default" },
    builtin = {
      title_fnamemodify = function(s) return s end,
  …
```

**Reply by mangelozzi** [reply]
Thanks so much!! I wonder what your day job is that you are able to offer such great support!  A great upgrade, and your dot files are a nice reference too, cheers!
